### PR TITLE
Change imported dep to match ROS 2 fork

### PR DESCRIPTION
**Pull request by Blast545**
As the title says.

Signed-off-by: Jorge Perez <jjperez@ekumenlabs.com>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -59,8 +59,8 @@ macro(build_mimick)
 
   include(ExternalProject)
   externalproject_add(mimick-ext
-    GIT_REPOSITORY https://github.com/hidmic/Mimick
-    GIT_TAG hidmic/cxx-compatibility
+    GIT_REPOSITORY https://github.com/ros2/Mimick
```
Comment (dirk-thomas) [inline CMakeLists.txt]: Nitpick: should have a `.git` suffix.

**Review by dirk-thomas — APPROVED** [review]

**Review by hidmic — APPROVED** [review]
LGTM pending green CI

**Comment by Blast545** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/11655/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/6781/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/9532/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/11629/)

**Comment by Blast545** [comment]
Another run after fixing Windows problem, CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/11659/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/6783/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/9534/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/11631/)

**Comment by Blast545** [comment]
All green, two reviews, merging.

**Pull request merged**